types/test/sortByProps.ts

```diff
@@ -0,0 +1,23 @@
+import * as RA from 'ramda-adjunct';
+
+const alice = {
+  name: 'ALICE',
+  age: 101,
+  lastName: 'Smith',
+};
+const bob = {
+  name: 'Bob',
+  age: -10,
+  lastName: 'Smith',
+};
+const clara = {
+  name: 'clara',
+  age: 314.159,
+  lastName: 'Kris',
+};
+const people = [clara, bob, alice];
+
+RA.sortByProps(people)(['name']); // $ExpectType object[]
```

Comment:
Parameters should be reversed and we're missing at least one test that is giving an error. Example: https://github.com/char0n/ramda-adjunct/blob/697fa142fde08a8d53ed2c3eda75ffe58ade217c/types/test/ceil.ts#L12